ecs-cli/modules/config/readwriter_v1_test.go

```diff
@@ -0,0 +1,341 @@
+	assert.NoError(t, err, "Error reading config")
+	_, ok := configMap[cfnStackNamePrefixKey]
+	assert.False(t, ok, "CFNStackNamePrefix should not exist in config")
+	_, ok = configMap[composeServiceNamePrefixKey]
+	assert.False(t, ok, "Compose service name prefix should not exist in config")
+	_, ok = configMap[composeProjectNamePrefixKey]
+	assert.False(t, ok, "Compose project name prefix should not exist in config")
+}
+
+func TestMissingPrefixesNewYAMLFormat(t *testing.T) {
+	configContentsNoPrefixes := `
+version: v0
+cluster: test
+aws_profile:
+region:us-west-2:
+aws_access_key_id:
+aws_secret_access_key:
+`
+	dest, err := newMockDestination()
+	assert.NoError(t, err, "Error creating mock config destination")
+
+	err = os.MkdirAll(dest.Path, *dest.Mode)
+	assert.NoError(t, err, "Could not create config directory")
+
+	defer os.RemoveAll(dest.Path)
+
+	err = ioutil.WriteFile(dest.Path+"/"+configFileName, []byte(configContentsNoPrefixes), *dest.Mode)
+	assert.NoError(t, err)
+
+	parser := setupParser(t, dest, true)
+	_, configMap, err := parser.GetConfig()
+	assert.NoError(t, err, "Error reading config")
+	_, ok := configMap[cfnStackNamePrefixKey]
+	assert.False(t, ok, "CFNStackNamePrefix should not exist in config")
+	_, ok = configMap[composeServiceNamePrefixKey]
+	assert.False(t, ok, "Compose service name prefix should not exist in config")
+	_, ok = configMap[composeProjectNamePrefixKey]
+	assert.False(t, ok, "Compose project name prefix should not exist in config")
+}
+
+func TestPrefixesDefaultNewYAMLFormat(t *testing.T) {
+	configContents := `
+version: v0
+cluster: test
+aws_profile:
+region: us-west-2
+aws_access_key_id:
+aws_secret_access_key:
+compose-project-name-prefix:
+compose-service-name-prefix:
+cfn-stack-name-prefix:
+`
+	dest, err := newMockDestination()
+	assert.NoError(t, err, "Error creating mock config destination")
+
+	err = os.MkdirAll(dest.Path, *dest.Mode)
+	assert.NoError(t, err, "Could not create config directory")
+
+	defer os.RemoveAll(dest.Path)
+
+	err = ioutil.WriteFile(configPath(dest), []byte(configContents), *dest.Mode)
+	assert.NoError(t, err)
+
+	parser := setupParser(t, dest, true)
+	readConfig, configMap, err := parser.GetConfig()
+	assert.NoError(t, err, "Error reading config")
+	_, ok := configMap[cfnStackNamePrefixKey]
+	assert.True(t, ok, "CFNStackNamePrefix should exist in config")
+	assert.Empty(t, readConfig.ComposeProjectNamePrefix, "Compose project prefix name should be empty.")
+	_, ok = configMap[composeServiceNamePrefixKey]
+	assert.True(t, ok, "Compose service name prefix should exist in config")
+	assert.Empty(t, readConfig.ComposeServiceNamePrefix, "Compose service prefix name should be empty.")
+	_, ok = configMap[composeProjectNamePrefixKey]
+	assert.True(t, ok, "Compose project name prefix should exist in config")
+	assert.Empty(t, readConfig.CFNStackNamePrefix, "CFNStackNamePrefix should be empty.")
+}
+
+func TestConfigFileTruncation(t *testing.T) {
```

Comment:
I don't think this test is needed since we are not writing config in ini formats.